### DISCUSSION TITLE
Heightmap plugin

### DISCUSSION THREAD

**Discussion by Valinakova** [Hoimar/Planet-Generator#14] [Ideas]
Hey, found this. Think there could be any use for it?  The license is appears to be extremely similar to the MIT license.

https://github.com/Zylann/godot_heightmap_plugin

**Comment by Valinakova** [comment]
I'm thinking of a feature where you can have a range of different environments for the planet. Like some parts of the planet where you'd have landscapes that look like this: 
![image](https://user-images.githubusercontent.com/81381626/120619157-9435b280-c410-11eb-8f74-6b5ffef013dc.png)

Areas where there could be plants like this and areas where they wouldn't.
![image](https://user-images.githubusercontent.com/81381626/120619271-aa437300-c410-11eb-9054-b4d0cd482a78.png)

The plugin also has it's own documentation https://hterrain-plugin.readthedocs.io/en/latest/

**Comment by Hoimar** [comment]
Thanks for the input, I know this plugin, it's really amazing at what it does. It does not work however on curved surfaces without major rework of the code. What I'd like to implement in the future though is the erosion landscape you posted in your second screenshot on the planets, and also different biomes (so icy, grassy, desert, etc.)
